src/super_gradients/recipes/quantization_params/default_quantization_params.yaml

```diff
@@ -0,0 +1,21 @@
+modify_recipe_for_qat:
+  enable: True                  # disable if you have recipe that is suitable for QAT
+  lr_decay_factor: 1e-2         # lr, weight decay, warmup and cosine parameters will be derived from original recipe with this factor
+  batch_size_divisor: 2         # (int) constant to divide batch_size if selective_quantizer_params.learn_amax=True
+  max_epochs_divisor: 10        # (int) constant to divide training_hyperparams.max_epochs for QAT
+  warmup_epochs_divisor: 10     # (int) constant to divide modified training_hyperparams.max_epochs to get training_hyperparams.lr_warmup_epochs
+  cosine_final_lr_ratio: 0.01   # (float) ratio between the end and the start of cosine lr schedule, overrides training_hyperparams.cosine_final_lr_ratio
+  disable_phase_callbacks: True # some phase callbacks interfere with QAT, this flag disables ALL phase callbacks
+
+selective_quantizer_params:
+  method_w: "max"           # calibrator type for weights, acceptable types are ["max", "histogram"]
+  method_i: "histogram"     # calibrator type for inputs acceptable types are ["max", "histogram"]
+  per_channel: True         # per-channel quantization of weights, activations stay per-tensor by default
+  learn_amax: False         # enable learnable amax in all TensorQuantizers using straight-through estimator
+  skip_modules: null        # optional list of module names (strings) to skip from quantization
+
+calib_params:
+  calib_method: "percentile"  # calibration method for all "histogram" calibrators, acceptable types are ["percentile", "entropy", mse"], "max" calibrators are not affected
```

Comment:
Its not so clear why there is method_w, method_i, learn_amax alongside with calib_params

Reply:
selective quantizer only adds calibrators for weights and inputs, which can be "max" and "histogram" but they don't know anything about calibration yet
Calibrator needs method names, which are ["max"] for "max" calibrator and ["mse", "entropy", "percentile"] for "histogram" one
Maybe, it's better to name it like "calib_i" and "calib_w" instead of "method_*" to make it less confusing

Reply:
But then doesn't it mean calib_method can be inferred from method_i and method_w ?

Is it useful to use different calibrators for inputs and weights ?

Reply:
calib_method is only relevant for "histogram" calibrators
"max" calibrators have only one option, which is "max" 

With selective quantizer we insert calibrators, which can only be of two types. With QuantizationCalibrator we call inserted calibrators, and for the histogram calibrator we have three different options ["mse", "entropy", "percentile"]